index.html

```diff
@@ -0,0 +1,174 @@
+            <li class="mission-card container">
+              <div class="mission-card-bg"></div>
+              <div class="flex-center content-mission">
+                <div class="icon flex-center">
+                  <img src="./img/clock-solid.svg" alt="" />
+                </div>
+                <div class="mission-title">
+                  <h2>Efficiency</h2>
+                </div>
+                <div class="mission-detail">
+                  <p>Rapidly Giving the Best</p>
+                </div>
+              </div>
+            </li>
+            <li class="mission-card container">
+              <div class="mission-card-bg"></div>
+              <div class="content-mission flex-center">
+                <div class="icon">
+                  <img src="./img/battery-full-solid.svg" alt="" />
+                </div>
+                <div class="mission-title">
+                  <h2>Maintainability</h2>
+                </div>
+                <div class="mission-detail">
+                  <p>Provide the best After Sales Service</p>
+                </div>
+              </div>
+            </li>
+            <li class="mission-card container">
+              <div class="mission-card-bg"></div>
+              <div class="content-mission flex-center">
+                <div class="icon">
+                  <img src="./img/check-circle-solid.svg" alt="" />
+                </div>
+                <div class="mission-title">
+                  <h2>Predictability</h2>
+                </div>
+                <div class="mission-detail">
+                  <p>
+                    Our devices know when they will be down, and they make you
+                    know beforehand!
+                  </p>
+                </div>
+              </div>
+            </li>
+
+            <li class="mission-btn">
+              <a href="#" class="btn">Become our partner 2022</a>
+            </li>
+          </ul>
+          <div class="see-more-container flex-center">
+            <a class="see-more-link" href="#">see more missions</a>
+          </div>
+        </div>
+      </section>
+      <section id="team" class="team">
+        <h2 class="team-title">OUR TEAM</h2>
+        <ul class="team-members container flex-center"></ul>
+        <div class="see-more">
+          <a href="#" class="btn container"
+            ><span>More</span> <span> <i class="fa fa-chevron-down"></i></span
+          ></a>
+        </div>
+      </section>
+      <section id="partner" class="our-partners">
+        <h2 class="about-title">Partner</h2>
+        <ul class="partner-list flex-center">
+          <li><img src="img/logo1.png" alt="partner image" /></li>
+          <li><img src="img/logo2.png" alt="partner image" /></li>
+          <li><img src="img/logo3.png" alt="partner image" /></li>
+          <li><img src="img/logo4.png" alt="partner image" /></li>
+          <li><img src="img/logo5.png" alt="partner image" /></li>
+        </ul>
+      </section>
```

Comment:
- [ ] This section also has different views for mobile and desktop screens. On desktop, it's supposed to be two elements side by side on each row, while on mobile they're all displayed in column. Please make sure to follow the design guidelines and make the required changes.

Reply:
Done